app/controllers/likes_controller.rb

```diff
@@ -7,7 +7,8 @@ def create
 
     if @like.save
       @like.update_likes_counter
-      redirect_to user_posts_path, notice: 'You liked this post!'
+      redirect_to user_posts_path
+      flash[:notice] = 'You liked this post!'
```

Comment:
- [ ] Flash messages do not work like expected. After like the post, the page redirects but to see the message ı have to go back to the posting page. Please fix it.

Reply:
Addressed